src/Pharo30Bootstrap/PBImageBuilder50.class.st

```diff
@@ -289,7 +289,7 @@ PBImageBuilder50 >> createInitialObjects [
 	self bootstrapInterpreter evaluateCode:
 		'Smalltalk globals at: #Transcript put: nil.'.
 	self bootstrapInterpreter evaluateCode:
-		'Smalltalk globals at: #SystemOrganisation put: nil.'.
+		'Smalltalk globals at: #SystemOrganization put: nil.'.
```

Comment:
I think you should just remove it since it worked without the correct name

Reply:
Using the right global seems to break the bootstrap anyway so let's remove it yeah